src/components/app-configure/ConfigureSidebar.vue

```diff
@@ -393,19 +431,16 @@ watch(
 }
 
 .sidebar__submenu-option {
+  position: relative;
   padding: 0 !important;
 }
 
-.sidebar__submenu-option:not(:last-child) {
-  margin-bottom: 1.5rem;
-}
-
 .sidebar__submenu-option-item {
   display: flex;
   flex-direction: row;
   gap: 12px;
   align-items: center;
-  justify-content: center;
+  justify-content: ;
```

Comment:
Can we remove this if this is not required?

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/295/commits/0d69c93b4cb98728da38188bdb528894405157d3)